deps/http_parser.mk

```diff
@@ -7,7 +7,7 @@ $(eval $(call git-external,http_parser,HTTPPARSER,,,$(BUILDDIR)))
 # Manually set the PLATFORM flag to whatever $(OS) is, because at least one
 # person (@staticfloat) has the PLATFORM envvar set to something else.
 # Also manually set PREFIX to DESTDIR
-HTTPPARSER_FLAGS := PLATFORM=$(shell echo $(OS) | tr A-Z a-z) PREFIX=$(build_prefix)
+HTTPPARSER_FLAGS := PLATFORM=$(shell echo $(OS) | tr A-Z a-z) PREFIX=$(build_prefix) LIBDIR=$(build_shlibdir)
```

Comment:
yes, if run in cygwin then ln -s creates a file that only cygwin can read. if run in msys, ln -s just copies i believe, so it's never really the right thing to do for mingw.